sequencer/src/lib.rs

```diff
@@ -312,8 +312,9 @@ where
     // Print the libp2p public key
     info!("Starting Libp2p with PeerID: {libp2p_public_key}");
 
+    let loaded_network_config_from_persistence = persistence.load_config().await?;
```

Comment:
This is a bit of a long name, but prevents rustfmt to reformat the entire let statement.

Reply:
I think here the intent is not to hold the read lock taken out in `load_config()` over the whole match statement so this change looks good to me. I'm not sure if this is what currently happens though because if yes, we should be observing a deadlock when we need the write lock to save the config later inside the match arm.

cc @imabdulbasit